x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/shared/hooks/metrics_to_api_options.test.ts

```diff
@@ -5,44 +5,59 @@
  * 2.0.
  */
 
-import type { MetricsMap } from './metrics_to_api_options';
+import type { MetricsExplorerOptions } from '../../../../pages/metrics/metrics_explorer/hooks/use_metrics_explorer_options';
+import { createMetricByFieldLookup, MetricsQueryOptions } from './metrics_to_api_options';
```

Comment:
Do we want to let the compiler auto-sort and deduplicate these imports?

Reply:
I was too optmistic that vscode would've added the new import to the existing list